### DISCUSSION TITLE
TableRepeater fields not stacking on mobile

### DISCUSSION THREAD

**Discussion by theo-1** [awcodes/filament-table-repeater#87] [Q&A]
This is what I have at 335px width in dev tools:

![nonstack](https://github.com/awcodes/filament-table-repeater/assets/71286048/53bddad4-b563-47f8-a6bd-e1b568c919d2)

There are further fields to the right completely clipped off.

Despite it being the default, I have still explicitly set `->breakpoint('md');`

What am I doing wrong?

**Comment by awcodes** [comment]
Make sure you added the views to your tailwind config and the css to your theme.css file then run a build.

**Reply by awcodes** [reply]
https://github.com/awcodes/filament-table-repeater#installation
